ush/python/pygw/src/pygw/exceptions.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=unused-argument
+
+# ----
+
+from collections.abc import Callable
+
+from pygw.logger import Logger, logit
+
+logger = Logger(level="error", colored_log=True)
+
+__all__ = ["Error", "msg_except_handle"]
+
+
+class Error(Exception):
```

Comment:
I would use a less generic class name here, like WorkflowError (or maybe WorkflowException if there is a desire to distinguish it as an abstract class from the concrete `*Error` classes).